packages/ai-lab/src/components/AILabImage/AILabImage.tsx

```diff
@@ -60,29 +80,31 @@ export const AILabImage = ({
 
     // https://arxiv.org/pdf/1704.04503.pdf, use Async to keep visuals
     const nmsDetections = await tf.image.nonMaxSuppressionWithScoreAsync(
-      justBoxes, // [numBoxes, 4]
-      justValues, // [numBoxes]
+      justBoxes,
+      justValues,
       maxBoxes,
       iouThreshold,
       detectionThreshold,
-      1 // 0 is normal NMS, 1 is Soft-NMS for overlapping support
+      nmsActive ? 1 : 0 // 0 is normal NMS, 1 is Soft-NMS for overlapping support
     );
 
     const chosen = await nmsDetections.selectedIndices.data();
     // Mega Clean
-    // tf.dispose([
-    //   results[0],
-    //   results[1],
-    //   model,
-    //   nmsDetections.selectedIndices,
-    //   nmsDetections.selectedScores,
-    //   prominentDetection.indices,
-    //   prominentDetection.values,
-    //   tensor,
-    //   readyfied,
-    //   justBoxes,
-    //   justValues,
-    // ]);
+    tf.dispose([
```

Comment:
I'm not sure why this dispose was commented out, but it needs to be brought back in to protect from mem-leaks

Reply:
You are right! I was getting type errors and since fast refresh disposed tensors I commented those out then forgot to brought them back even after the type errors are fixed in different components.